sample-conference-videochat-swift/sample-conference-videochat-swift/sample-conference-videochat-swift/Components/Auth/ViewControllers/AuthViewController/LoginTableViewController.swift

```diff
@@ -8,48 +8,201 @@
 
 import UIKit
 
+struct LoginConstant {
+    static let enterToChat = NSLocalizedString("Enter to chat", comment: "")
+    static let login = NSLocalizedString("Login", comment: "")
+    static let checkInternet = NSLocalizedString("Please check your Internet connection", comment: "")
+    static let enterUsername = NSLocalizedString("Please enter your username and group name. You can join existent group.", comment: "")
+    static let shouldContainAlphanumeric = NSLocalizedString("Field should contain alphanumeric characters only in a range 3 to 20. The first character must be a letter.", comment: "")
+    static let shouldContainAlphanumericWithoutSpace = NSLocalizedString("Field should contain alphanumeric characters only in a range 3 to 15, without space. The first character must be a letter.", comment: "")
+    static let showUsers = "ShowUsersViewController"
+    static let userNameRegex = "^[^_][\\w\\u00C0-\\u1FFF\\u2C00-\\uD7FF\\s]{2,19}$"
```

Comment:
Move regular extentions to LoginRegularExtention struct:
struct LoginNameRegularExtention {
    static let user = "^[^_][\\w\\u00C0-\\u1FFF\\u2C00-\\uD7FF\\s]{2,19}$"
    static let chat = "^[a-zA-Z][a-zA-Z0-9]{2,14}$"
}